src/app/admin/releasePage/releasePage.component.html

```diff
@@ -196,27 +196,27 @@ <h2>Release Management</h2>
           <h2>FAQ</h2>
           <ul>
             <li>
-              <strong>Q. Icon is Missing</strong>
-              <p>A. Make sure it is published and not in a draft state.</p>
+              <strong>Icon is Missing</strong>
+              <p>Make sure it is published and not in a draft state.</p>
             </li>
             <li>
-              <strong>Q. What is a codepoint?</strong>
-              <p>A. A codepoints is a index (hex encoded) in a font. These must be unique within a single font and should not change between releases.</p>
+              <strong>What is a codepoint?</strong>
+              <p>A codepoint is an index (hex encoded) in a font. Codepoints must be unique within a single font and should not change between releases.</p>
             </li>
             <li>
-              <strong>Q. How do I know what codepoint to use?</strong>
-              <p>A. Assuming you want the next in the series simply click in a blank input.</p>
+              <strong>How do I know what codepoint to use?</strong>
+              <p>Assuming you want the next in the series simply click in a blank input.</p>
```

Comment:
What does "simply click in a blank input" mean? Place the cursor in the empty textbox for the codepoint to auto-populate it?